pages/query-keys.ts

```diff
@@ -1,12 +1,19 @@
 //descriptive names: Something like 'GET_COMMENTS', 'GET_COMMENT', 'ADD_USER'
-
-
-
 //I think these should be all caps, the values. Because they are constants, and in JS world, we use CAPS_CAMEL_CASE to represent constants
 
-const queryKeys = {
+type QueryKey = {
+  FETCH_COMMENTS: string;
+  FETCH_USERS: string;
+  FETCH_NEWS: string;
+};
```

Comment:
This type is not so necessary ... Typescript would infer the type of queryKeys as `Record<string, string>`